### ISSUE TITLE
solve the errors during compiling darknet under linux withoud opencv

### ISSUE THREAD

**Issue by yjphhw** [pjreddie/darknet#1684]
during compile the darknet, the error says can't create image_opencv.o 

I look through makefile find that in below clause:
OBJ=gemm.o utils.o cuda.o deconvolutional_layer.o convolutional_layer.o list.o image.o activations.o im2col.o col2im.o blas.o crop_layer.o dropout_layer.o maxpool_layer.o softmax_layer.o data.o matrix.o network.o connected_layer.o cost_layer.o parser.o option_list.o detection_layer.o route_layer.o upsample_layer.o box.o normalization_layer.o avgpool_layer.o layer.o local_layer.o shortcut_layer.o logistic_layer.o activation_layer.o rnn_layer.o gru_layer.o crnn_layer.o demo.o batchnorm_layer.o region_layer.o reorg_layer.o tree.o lstm_layer.o l2norm_layer.o yolo_layer.o iseg_layer.o image_opencv.o

the last file name is image_opencv.o  
the author said he used stb lib for image IO, so it seems useless when not include 
I tried delete the string "image_opencv.o" 
then I compiled, and got succeed.  @pjreddie
